src/app/component/general/eco-news/create-news/create-news.component.css

```diff
@@ -1,31 +1,150 @@
-a {
-    text-decoration: none;
```

Comment:
General: all fonts should be implemented according to mockup

Reply:
fixed